### PR TITLE
Online booking demand experiment

### PR DESCRIPTION
**Pull request by stevenwilkin**

### CODE REVIEW

**Comment by andrewgarner** [comment]
This is ready for review @guidance-guarantee-programme/developers

File: bower.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "name": "pension_guidance",
   "private": true,
   "dependencies": {
-    "govuk_elements": "https://github.com/alphagov/govuk_elements.git#1e4f3d5e0cab2d7dc7e350190ab7b1c32a1f2aa9",
+    "govuk_elements": "https://github.com/alphagov/govuk_elements.git",
```
Comment (stevenwilkin) [inline bower.json]: This can be put directly onto master?

File: lib/appointment_ticket.rb [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+  def info
+    case @appointment.type
+    when 'phone'
+      <<-TEXT.strip_heredoc
+        Memorable word:
+        #{@appointment.memorable_word}
+      TEXT
+    when 'face-to-face'
+      <<-TEXT.strip_heredoc
+        Post code:
```
Comment (stevenwilkin) [inline lib/appointment_ticket.rb]: Postcode is a single word

File: app/decorators/appointment_decorator.rb [hunk 3]
```diff
@@ -0,0 +1,15 @@
+class AppointmentDecorator < Draper::Decorator
+  delegate_all
+
+  def date
+    @date ||= object.slot.strftime('%A, %-d %B %Y')
```
Comment (stevenwilkin) [inline app/decorators/appointment_decorator.rb]: Do we really need to memoize the values computed in these methods?

File: app/views/appointments/new.html.erb [hunk 4]
```diff
@@ -0,0 +1,32 @@
+<% content_for(:page_title, t('service.title', page_title: 'Appointment times')) %>
+
+<% content_for :head do %>
+  <style>
```
Comment (stevenwilkin) [inline app/views/appointments/new.html.erb]: Are we ok with putting CSS directly into the view?

File: app/views/styleguide/pages/appointment_details.erb [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+
+<h1>Appointment details</h1>
+
+<p>Date<br>
+  <strong>Monday 27 July 2015</strong></p>
+
+<p>Time<br>
+  <strong> 8:30 am</strong></p>
+
+<form class="new_appointment" id="new_appointment" action="/appointments" accept-charset="UTF-8" method="post"><input name="utf8" type="hidden" value="✓"><input type="hidden" name="authenticity_token" value="KAW5TqtGIfoKRG4ppn2UeCHYxvDByMUBbGX28lvZ320oMSn5ngeK5OnkHhFR9bvv33iEfXpxv29WydPrsWRH3w==">
```
Comment (stevenwilkin) [inline app/views/styleguide/pages/appointment_details.erb]: Hardcoded authentication_token, any need for this? Does it work?

File: app/views/appointments/new.html.erb (hunk 4 above)
Comment (aduggin) [inline app/views/appointments/new.html.erb]: We are for this 1 day experiment

Comment (stevenwilkin) [inline app/views/appointments/new.html.erb]: :tophat:

File: app/views/styleguide/pages/appointment_details.erb (hunk 5 above)
Comment (aduggin) [inline app/views/styleguide/pages/appointment_details.erb]: Not needed. Removed.

File: bower.json (hunk 1 above)
Comment (aduggin) [inline bower.json]: We can - but I'l do more testing before I do that.

**Pull request merged**